pipeline/src/transformers/addressables.ts

```diff
@@ -59,9 +64,10 @@ export const transformAddressable = (
       transformedDocument = transformAddressableExhibitionText(document);
       break;
 
-    // case 'exhibitions-highlight-tours':
-    //   transformedDocument = transformAddressableBook(document);
-    //   break;
+    case 'exhibition-highlight-tours':
+      transformedDocument =
+        transformAddressableExhibitionHighlightTour(document);
+      break;
```

Comment:
Did we figure out how to pass in an array, is it alright to do? We might have to change this slightly? https://github.com/wellcomecollection/content-api/blob/main/pipeline/src/extractTransformLoad.ts

Reply:
I wasn't sure what was required, but I mostly ignored it once I got the tests passing. Maybe we can go through this together?

Reply:
Let's talk about it at the FE catchup!